packages/coinjoin/src/backend/scanAccount.ts

```diff
@@ -117,11 +118,14 @@ export const scanAccount = async (
         }
     }
 
-    await mempool.update();
+    let pending: Transaction[] = [];
```

Comment:
would need to be `ReturnType<ReturnType<typeof transformTx>>[]` since `transformTx` only returns parametrized call to `transformTransaction` imported from  `@trezor/blockchain-link/lib/workers/blockbook/utils`

Reply:
but I understand that it would be nice to limit imports from blockchain-link if possible :thinking: